### DISCUSSION TITLE
Missing package folder after run Custom build

### DISCUSSION THREAD

**Discussion by notrealdev** [nolimits4web/swiper#5018] [Q&A] [answered]
Hi, I am trying to create a custom build for my website, follow the document [here](https://swiperjs.com/swiper-api#using-build-script).
When I run `npm run build:prod`, nothing happens, the `package` folder is not created.

Check my image:
![Imgur](https://i.imgur.com/EHUT21b.jpg)

Did I make a mistake or mistake somewhere?
Can anyone help me to solve the problem?

Thank!

**Comment by nolimits4web** [comment, marked as answer]
It is in the `dist` folder

**Reply by notrealdev** [reply]
I have seen, it is ignored in the file `.gitignore`
Should you also update the document?
Thanks!
